pkgs/applications/networking/p2p/qbittorrent/default.nix

```diff
@@ -95,6 +91,13 @@ stdenv.mkDerivation rec {
     changelog = "https://github.com/qbittorrent/qBittorrent/blob/release-${version}/Changelog";
     license = licenses.gpl2Plus;
     platforms = platforms.unix;
-    maintainers = with maintainers; [ Anton-Latukha kashw2 ];
+    maintainers = with maintainers; [
+      Anton-Latukha
+      kashw2
+    ];
+    mainProgram =
+      "qbittorrent"
+      + lib.optionalString (guiSupport && qtVersion == "5") "-qt5"
+      + lib.optionalString (!guiSupport) "-nox";
```

Comment:
This is equal to cfg.pname, right? Any reason to not use that?

Reply:
As I've had it described to me, this is an anti pattern.

However yes it would be.